### PR TITLE
[WIP] Remove dependency on symfony/templating

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because this is BC break.

## Changelog

```markdown
### Changed
- Removed dependency on `symfony/templating`
```

## To do

- [x] Remove dev repository from composer.json after https://github.com/sonata-project/SonataBlockBundle/pull/476 will be merged

## Subject

This PR is a part of a plan of decoupling sonata-project bundles from Templating Component. See https://github.com/sonata-project/dev-kit/issues/390 for details

### CODE REVIEW

File: src/Block/AdminStatsBlockService.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ class AdminStatsBlockService extends AbstractBlockService
     /**
      * @param string $name
```
Comment (kunicmarko20) [inline src/Block/AdminStatsBlockService.php]: you can remove this and add `string` typehint to name

Comment (covex-nn) [reply src/Block/AdminStatsBlockService.php]: Sure. Then i will do it for `AbstractBlockService::__construct` in BlockBundle first

**Comment by OskarStark** [comment]
sonata-project/SonataBlockBundle#476 is merged, I asked for a release

**Comment by covex-nn** [comment]
@OskarStark i guess, it is too early for major release. PR for BlockBundle was merged into `master`, and this PR is for `master` branch too.

**Comment by OskarStark** [comment]
> @OskarStark i guess, it is too early for major release. PR for BlockBundle was merged into master, and this PR is for master branch too.

you are right, I missed, that both of them are against `master` 😄

**Comment by OskarStark** [comment]
Could you finalize this PR?

**Comment by covex-nn** [comment]
@OskarStark i'm waiting for https://github.com/sonata-project/dev-kit/pull/400 (it is a blocker for this PR)

**Comment by core23** [comment]
This PR looks dead. Feel free to reopen if you will continue working on this.

**Pull request closed without merging**